cmake/adios2-config-common.cmake.in

```diff
@@ -12,6 +12,11 @@ endif()
 
 set(ADIOS2_HAVE_Fortran @ADIOS2_HAVE_Fortran@)
 set(ADIOS2_HAVE_MPI @ADIOS2_HAVE_MPI@)
+set(ADIOS2_HAVE_CUDA @ADIOS2_HAVE_CUDA@)
+set(ADIOS2_HAVE_Kokkos @ADIOS2_HAVE_Kokkos@)
```

Comment:
Shouldn't we also have:
```
  if(ADIOS2_HAVE_Kokkos)
    find_dependency(Kokkos)
  endif()
```

Reply:
the adios2_core_kokkos target has a private dependency to Kokkos::kokkos. I think that in that case we do not need to load the kokkos targets (use find_dependency) since the private dependencies are not exported in a installation

Reply:
Meaning that is on the user to do the `find_package(kokkos)` if the want to pass a kokkos_view to adios2.